picard/tagger.py

```diff
@@ -412,7 +412,11 @@ def process(result=None, error=None):
                     self.add_files(result)
                 thread.run_task(get_files, process)
 
-        process(True, False)
+        if config.setting['recursively_add_files']:
+            process(True, False)
+        else:
+            self.add_files(os.path.join(path, f) for f in os.listdir(path))
```

Comment:
Calling `add_files` here creates a new codepath without all the statusbar and debugging messages that `get_files` has, which is not good.